prime-router/pom.xml

```diff
@@ -317,7 +317,7 @@
                             <pojos>true</pojos>
                             <pojosEqualsAndHashCode>true</pojosEqualsAndHashCode>
                             <pojosToString>true</pojosToString>
-                            <immutablePojos>true</immutablePojos>
+                            <immutablePojos>false</immutablePojos>
                         </generate>
```

Comment:
You'll see in the ActionHistory container class, that I'm using the generated POJOs as convenient storage boxes.  For this to work, I needed the POJOS to be mutable.